config/status-crd.yaml

```diff
@@ -0,0 +1,34 @@
+kind: CustomResourceDefinition
```

Comment:
this can point to https://github.com/openshift/machine-api-operator/blob/master/config/0000_00_cluster-version-operator_01_clusteroperator.crd.yaml intead